lib/partitions_validator_utils.pm

```diff
@@ -24,4 +30,64 @@ sub validate_partition_table {
     assert_equals($args->{table_type}, $table_type, "Partition table type does not correspond to the expected one.");
 }
 
+sub validate_partition_table {
+    my ($args) = @_;
+    return if check_var('BACKEND', 's390x');                                       # fdisk output does not show partition table for dasd
+    record_info("Check $args->{table_type}", "Verify that partition table type is $args->{table_type}");
+    my $table_type = (split(/\"/, script_output("blkid $args->{device}")))[-1];
+    assert_equals($args->{table_type}, $table_type, "Partition table type does not correspond to the expected one.");
+}
+
+sub validate_partition_creation {
+    my ($args) = @_;
+    my @lsblk_output = split(/\n/, script_output("lsblk -n "));
+    record_info("Check $args->{mount_point}", "Verify that partition $args->{mount_point} was created.");
+    my $check = 0;
+    if ($args->{mount_point} eq 'swap') {
+        foreach (@lsblk_output) {
+            if ($_ =~ /\[SWAP\]/) {
+                $check = 1;
+            }
+            last if ($check == 1);
+        }
+    }
+    else {
+        foreach (@lsblk_output) {
+            if ($_ =~ /$args->{mount_point}\z/) {
+                $check = 1;
+            }
+            last if ($check == 1);
+        }
+    }
+    assert_equals($check, 1, "The $args->{mount_point} partition was not created.");
+}
+
+sub validate_filesystem {
+    my ($args) = @_;
+
+    record_info("Check filesystem", "Verify that $args->{mount_point} partition filesystem is $args->{fs_type}");
+    my @df_output = split(/\n/, script_output("df -T $args->{mount_point}"));
+    my $type      = ((split(/\s*\s/, $df_output[1])))[1];
```

Comment:
just `split(' ',...`
I would use `shift $df_output` to avoid at all cost indexes, but it is fine, I think it would be good to put in a different line that operation so we indicate that we are discarding the column names row.
Same for the second index, another comment would be helpful, that you are getting the type.
In general for the future we can use some hash to parse this, so you would need less comments, basically you could have a hash that has those column names, `$df_output->{Type}` and then is more clear that the indexes, but just for next time.

Reply:
I assumed you preferred not change it, which is fine, but might at least ask you for adding a comment?

Reply:
Ah, ok sorry. I don't know why, '\n' seems a bit more clear to me when I read code instead of ' '. That's why I didn't change it.

Reply:
I was referring to the line above my comment, with [1] and [1], but not big deal.